irods/test/admin_test.py

```diff
@@ -91,6 +91,10 @@ def test_groupadmin_creates_user_group__374(self):
             group = bobby.user_groups.create(user_group_name)
             self.assertEqual(group.name, user_group_name)
 
+            # groupadmin cannot remove groups!
+            with self.assertRaises(SYS_NO_API_PRIV):
+                group.remove()
+
```

Comment:
Is it worth having a dedicated test for this?

I'm thinking about the name of this test and how it does not convey anything about this assertion.

Reply:
easy fix - rename the test?!

Reply:
Very true.